src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -1075,18 +1075,25 @@ public String getColumnFamily() {
 
   private void registerConnectionsGauge() {
     try {
-      if (!this.metricRegistry
+      if (!metricRegistry
           .getGauges()
           .containsKey(
               MetricRegistry.name(
-                  JmxProxyImpl.class, this.clusterName, this.host, "repairStatusHandlers"))) {
-        this.metricRegistry.register(
+                  JmxProxyImpl.class,
+                  clusterName.replaceAll("[^A-Za-z0-9]", ""),
```

Comment:
You'll remove any dash or underscore that could exist in the cluster name. This could make it harder for peeps to match their cluster names automatically.

Reply:
it was @joaquin that reported that cluster names with `.` could get into the metric names.
so reaper doesn't sanitise them enough…

maybe it should just be `clusterName.replace('.', '-')` instead of `clusterName.replaceAll("[^A-Za-z0-9]", "")`

Reply:
yes it would be better IMHO